tests/PySys/tedge_agent/agent_init_session/run.py

```diff
@@ -0,0 +1,73 @@
+from pysys.basetest import BaseTest
+from datetime import datetime, timedelta
+
+"""
+Validate tedge agent init session feature.
+
+
+Given unconnected system
+
+When we start tedge_agent to initialize session and exit
+When we start publish the software list request
+When we start a subsciber to get the response for the request
+When we start the tedge_agent, it gets the previous response and responds
+Then stop the agent and the subscriber
+Then validate the output of the subscriber for response
+"""
+
+class AgentInitSession(BaseTest):
+    def setup(self):
+        self.sudo = "/usr/bin/sudo"
+        self.tedge_agent = "/usr/bin/tedge_agent"
+        self.tedge = "/usr/bin/tedge"
+        self.systemctl = "/usr/bin/systemctl"
+
+        remove_lock = self.startProcess(
+            command=self.sudo,
+            arguments=["rm", "-rf", "/var/lock/tedge_agent.lock"],
+            stdouterr="remove_lock",           
+        )    
+
+    def execute(self):        
+        agent_init = self.startProcess(
+            command=self.sudo,
+            arguments=[self.tedge_agent, "--init"],
```

Comment:
It might be good to run a `--clean` before `--init`.

Reply:
done